### PR TITLE
clarify ola_set_dmx help

### LINKED ISSUE #1109: ola_set_dmx documentation
Hi,

I received the [following](https://bugs.debian.org/830972) bug report at Debian:

```
there is no manpage for ola_set_dmx, and the brief help text basically says:

 -d, --dmx <values>        Comma separated DMX values.

What does that mean? Should I enter "-d Channel,Value,Channel2,Value2, ..."
Or should I enter "-d Channel0,Channel1,Channel2, ... Channel512",
or is it something completely different? 

To add insult to injury, googling does not bring up much more than
people using 255,0,255,0 or similar for testing. But what does this do?

Especially when starting out and testing hardware of unknown working order
a more explicit description would be helpful.
```

### PR DESCRIPTION
**Pull request by FloEdelmann**
Added examples to clarify usage of `ola_set_dmx`. Closes #1109.

How should I update the [man page](https://github.com/OpenLightingProject/ola/blob/master/man/ola_set_dmx.1#L1), as it says `DO NOT MODIFY THIS FILE!  It was generated by help2man 1.37.1.`?

### CODE REVIEW

**Comment by peternewman** [comment]
Can you retarget this at our 0.10 branch please.

Info on the help2man here:
https://github.com/OpenLightingProject/ola/issues/940

I think I added info to quite a few of them, hence it's not fully automated via help2man (plus as we migrate them to our flags module, this becomes unnecessary (for the C++ at least)).

File: examples/ola-client.cpp [hunk 1]
```diff
@@ -656,13 +656,14 @@ void DisplayUniverseMergeHelp(const options &opts) {
...
+  cout << "Usage: " << opts.cmd << " --universe <universe> --dmx <values>\n"
           "\n"
           "Sets the DMX values for a universe.\n"
           "\n"
           "  -h, --help                Display this help message and exit.\n"
-          "  -u, --universe <universe> Universe number.\n"
-          "  -d, --dmx <values>        Comma separated DMX values.\n"
+          "  -u, --universe <universe> Universe number, e.g. 0.\n"
+          "  -d, --dmx <values>        Comma separated DMX values, e.g. "
+          "0,255,128 sets channel 1 to 0, channel 2 to 255, channel 3 to 128.\n"
```
Comment (peternewman) [inline examples/ola-client.cpp]: I'd be tempted to label these as the first, second and third channels, to avoid the whole counting from 0 or 1 issue.

Comment (FloEdelmann) [inline examples/ola-client.cpp]: Good idea!

**Comment by FloEdelmann** [comment]
I guess that's not how you do a proper rebase...

Should I open a new clean pull request?

BTW, thanks @peternewman for inviting me to the project :)

**Comment by peternewman** [comment]
Your welcome @FloEdelmann . It means we can assign stuff to you in the issue list now. :)

**Pull request closed without merging**